src/components/Search/EditButton.tsx

```diff
@@ -1,25 +1,27 @@
 import Button from '@mui/material/Button';
 
 import { Strings } from '../../resources/Strings';
+import pencilDark from '../../theme/img/pencil-dark.svg';
+import pencil from '../../theme/img/pencil.svg';
 
 interface EditButtonProps {
   onEditAction: () => void;
+  mode: string;
 }
 
 const baseComp = Strings.components.searchDefault.base;
-const editImgUrl = baseComp.editIcon;
```

Comment:
shouldn't that be removed from the Strings object now?

Reply:
Right!